connect/runtime/src/test/java/org/apache/kafka/connect/runtime/standalone/StandaloneHerderTest.java

```diff
@@ -262,44 +234,15 @@ public void testRestartTask() throws Exception {
     }
 
     @Test
-    public void testRestartTaskFailureOnStop() throws Exception {
```

Comment:
I'm guessing you got rid of these because we catch exceptions thrown in stop() inside `WorkerConnector` and `WorkerTask`. Is that right?